.travis.yml

```diff
@@ -23,6 +23,7 @@ matrix:
 
 env:
   global:
+    - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml" SYMFONY_PHPUNIT_VERSION=5.7
```

Comment:
Is this allowed, all on one line?

Reply:
afaik that works, yes. probably because this is how environment variables are passed anyways to the scripts that are run - travis-ci likely just concats the rows with whitespace separator.